examples/medusa/convert_checkpoint.py

```diff
@@ -1188,8 +1188,20 @@ def load_medusa_hf(medusa_path: str,
                                    mapping=Mapping(),
                                    dtype='float32'):
                     logger.info("Loading Medusa heads' weights ...")
+                    is_ckpt_safetensors = False
+
                     ckpt_file = Path(medusa_path) / "medusa_lm_head.pt"
-                    state_dict = torch.load(ckpt_file, map_location="cpu")
+                    if not ckpt_file.exists():
+                        ckpt_file = Path(medusa_path) / "medusa_lm_head.safetensors"
+                        is_ckpt_safetensors = True
+                    
+                    if is_ckpt_safetensors:
+                        print("Safetensors Found ...")
```

Comment:
Could you change the print to logger.info?

Reply:
Done